server/templates/staff/course/assignment/assignment.new.html

```diff
@@ -25,33 +25,62 @@
+                <h3>Name</h3>
+                <fieldset>
+                  {{ forms.render_field(form.display_name, label_visible=true, placeholder='Hog', required="required", type='text') }}
                   {{ forms.render_field(form.name, label_visible=true, value=current_course.offering + '/', placeholder='cal/cs61a/fa16/proj01', type='text') }}
                   {{ forms.render_field(form.max_group_size, label_visible=true, type='number', min='1', value='1') }}
-                  {{ forms.render_field(form.due_date, label_visible=true, value=utils.new_due_date(current_course), type='text', class='form-control datepicker') }}
-                  {{ forms.render_field(form.lock_date, label_visible=true, value=utils.new_lock_date(current_course), type='text', class='form-control datepicker') }}
-                  {{ forms.render_field(form.url, label_visible=true, placeholder='http://cs61a.org/proj/hog/ (Optional)', type='text') }}
-                  {{ forms.render_checkbox_field(form.visible, label_visible=true) }}
-                  {{ forms.render_field(form.autograding_key, label_visible=true, placeholder='5abc... (Optional)', type='text') }}
-                  {{ forms.render_checkbox_field(form.revisions_allowed, label_visible=true) }}
-                  {{ forms.render_checkbox_field(form.uploads_enabled, label_visible=true) }}
-                  {{ forms.render_field(form.upload_info, label_visible=true, placeholder='Upload the following files: test.py', type='text') }}
+                </fieldset>
+                <h3>Deadlines</h3>
+                <fieldset>
+                    {{ forms.render_field(form.due_date, label_visible=true, value=utils.new_due_date(current_course), type='text', class='form-control datepicker') }}
+                    {{ forms.render_field(form.lock_date, label_visible=true, value=utils.new_lock_date(current_course), type='text', class='form-control datepicker') }}
+                </fieldset>
+                <h3>Grading</h3>
+                <fieldset>
+                    {{ forms.render_checkbox_field(form.visible, label_visible=true) }}
+                    {{ forms.render_field(form.autograding_key, label_visible=true, placeholder='5abc... (Optional)', type='text') }}
+                    {{ forms.render_checkbox_field(form.revisions_allowed, label_visible=true) }}
+                    {{ forms.render_checkbox_field(form.uploads_enabled, label_visible=true) }}
+                    {{ forms.render_field(form.upload_info, label_visible=true, placeholder='Upload the following files: test.py', type='text') }}
+                </fieldset>
               {% endcall %}
             </div>
           </div>
           <!-- /.box-body -->
         </div>
         <!-- /.box -->
       </div>
+  <!-- </body> do not close body in template-->
+{% endblock %}
 
+{% block page_js %}
+<script type="text/javascript"  src="https://cdnjs.cloudflare.com/ajax/libs/jquery-steps/1.1.0/jquery.steps.js"></script>
```

Comment:
This is ... questionable  - but since we only use it once - I didn't want to bother with adding to all pages. 

I really want to write a flask-assets/webassets resolvers for URLs